src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -1205,7 +1205,7 @@ void ApplicationManagerImpl::OnStreamingConfigured(
       NaviServiceStatusMap::iterator it = navi_service_status_.find(app_id);
       if (navi_service_status_.end() == it) {
         LOG4CXX_WARN(logger_, "Application not found in navi status map");
-        connection_handler().NotifyServiceStartedResult(false, empty);
+        connection_handler().NotifyServiceStartedResult(app_id, false, empty);
```

Comment:
My apologies if you've looked deeper into this than I have, but the internal `app_id` isn't the same as the `session_key`, correct? They are used interchangeably for this function.